Sources/SRGDataProviderModel/include/SRGTypes.h

```diff
@@ -665,15 +665,12 @@ typedef NS_CLOSED_ENUM(NSInteger, SRGContentProviders) {
 };
 
 /**
- *  Image scalings.
- */
-typedef NS_CLOSED_ENUM(NSInteger, SRGImageScaling) {
-    SRGImageScalingAspectFillSixteenToNine = 0,
-    SRGImageScalingDefault = SRGImageScalingAspectFillSixteenToNine,
-    SRGImageScalingAspectFitSixteenToNine,
-    SRGImageScalingAspectFitBlackSquare,
-    SRGImageScalingAspectFitTransparentSquare,
-    SRGImageScalingPreserveAspectRatio
+ *  Image scaling services.
+ */
+typedef NS_CLOSED_ENUM(NSInteger, SRGImageScalingService) {
+    SRGImageScalingServiceBusinessUnit = 0,
+    SRGImageScalingServiceDefault = SRGImageScalingServiceBusinessUnit,
```

Comment:
I agree. To get more transparence, remove the `default` state and have only two.

Regarding a boolean, it would be great to validate with Android team as well if `centralized` is correctly named?

 - `SRGImageScalingServiceBusinessUnit` is business unit responsibility: it uses their services on their domain.
 - `SRGImageScalingServiceCentralized` is Play SRG responsibility: it used the "new image service" on ìl.srgssr.ch` domain.